internal/donor/donorpage/task_list.go

```diff
@@ -158,6 +158,8 @@ func taskListPaymentSection(provided *donordata.Provided) taskListSection {
 		paymentPath = donor.PathPayFee.Format(provided.LpaID)
 	case task.PaymentStateMoreEvidenceRequired:
 		paymentPath = donor.PathUploadEvidence.Format(provided.LpaID)
+	case task.PaymentStatePending, task.PaymentStateCompleted:
```

Comment:
Not part of the ticket but this felt like a sensible change to prevent duplicate payments.